CHANGELOG.md

```diff
@@ -0,0 +1,168 @@
+﻿# Change Log
```

Comment:
What are changes in the coming release?

Reply:
Changelog requires **version number** and **date** of the release. So it will be better to update the changelog at the time of release.